composer.json

```diff
@@ -6,8 +6,8 @@
   "require": {
     "php": ">=5.6",
     "composer/installers": "^v1.12.0 || ^2.2",
-    "dealerdirect/phpcodesniffer-composer-installer": "^0.4.1 || ^0.5 || ^0.6.2 || ^0.7 || ^1.0",
-    "wp-coding-standards/wpcs": "^2.3 | dev-develop#fca9d9e",
+    "dealerdirect/phpcodesniffer-composer-installer": "^1.0.0",
+    "wp-coding-standards/wpcs": "dev-develop#fca9d9e as 2.3.0",
     "automattic/vipwpcs": "^2.0.0"
```

Comment:
We should remove below rule from VIP https://github.com/Automattic/VIP-Coding-Standards/blob/develop/WordPressVIPMinimum/ruleset.xml#L29-L30

```
<rule ref="WordPress.PHP.StrictComparisons"/>
<rule ref="WordPress.CodeAnalysis.AssignmentInCondition"/>
```

Reply:
@mukeshpanchal27 How would we do that and why should we do that?

Reply:
@spacedmonkey because we use WPCS development version (3.X) and they Replace this dependancies

- `WordPress.PHP.StrictComparisons` sniff in https://github.com/WordPress/WordPress-Coding-Standards/pull/1919
- `WordPress.CodeAnalysis.AssignmentInCondition` sniff in https://github.com/WordPress/WordPress-Coding-Standards/pull/2198

I tried several thing to exclude that but no luck. Do you know how we can do that?

Reply:
Fixed.